src/Taskview/SearchBar.ts

```diff
@@ -300,10 +300,11 @@ export class SearchBar {
         if (d.optionType === 'gene') {
           // set global optionId
           this._geneHoverOptionId = d.optionId;
+          const currentTarget = event.currentTarget as HTMLElement; // store current target as it becomes null by the time the timeout is executed
           setTimeout(() => {
             // update detail after timeout time global and current optionId is equal
             if (d.optionId === this._geneHoverOptionId) {
-              this._mouseOverHandler(d, event.currentTarget);
+              this._mouseOverHandler(d, currentTarget);
```

Comment:
It might not be necessary, but you can also use it on [line 311](https://github.com/Caleydo/coral/pull/66/files#diff-2b469dc79c3c9df4c513d6282a7a22d8b4773f69920f7382f5d777fa5b23aba9R311).

Reply:
Line 311 only handles the case for non-genes. Ollie fixed the gene case here and the event might not be available after the timeout is executed. So I agree with the fix he proposed.